components/Documentbar/Documentbar.context.tsx

```diff
@@ -9,7 +9,7 @@ import { DocumentbarInitialState } from './Documentbar.state';
 export interface DocumentbarContextProps {
   state: DocumentbarInitialState;
   dispatch: Dispatch<ActionType<DocumentbarInitialState>>;
-  handleCreateDocument: () => void;
+  handleCreateDocument: (event: ChangeEvent<HTMLInputElement>) => void;
```

Comment:
do you understand what is a `ChangeEvent`? My first time seeing so I might need to read up on it

Reply:
the event is a file upload from a form